### ISSUE TITLE
Editorial: Add image description text for the audio splice rendering image (in current WD section 4.5.12)

### ISSUE THREAD

**Issue by wolenetz** [w3c/media-source#312]
This issue tracks the following portion of #307:

> §3.5.12 - The image representing the algorithm lacks a description. It's acknowledged that the image is itself a graphical representation of the algorithm described above it, but looks like it's a particularly concrete example, and it has specific features that should be described to help people interpret it. Another way to phrase this would be that the image isn't what I had in mind when reading the algorithm (though that may be due to lack of experience). For example, there are vertical lines and gradients at specific times in the image. It seems like a separate description may be warranted.

Note that this section, in the current WD, is now 4.5.12, and may change. It is the image example of the audio splice rendering algorithm.

Note also that the reference to external descriptive text (like the pipeline model image description) may need to refer directly to the ED version since it is not auto-published. See also #308 and #311, and consult @tidoust for best way to add this so both ED and WD get it.

**Comment by chrisn** [comment]
The image is now in section 5.5.12.
